.circleci/config.yml

```diff
@@ -13,6 +13,8 @@ jobs:
        - run: ci/do_circle_ci.sh bazel.release
        - setup_remote_docker
        - run: ci/docker_push.sh
+       - deploy:
```

Comment:
any reason to use deploy here vs. run?

Reply:
deploy was what the examples used, but run will work and read better.